cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -479,7 +479,11 @@ var (
 				logger.Warn("No fields selected, won't update anything")
 				return nil
 			}
-			var device ttnpb.EndDevice
+			device := ttnpb.EndDevice{
```

Comment:
I don't think this is the right place for this. Setting the "fallback" addresses from config (if still zero in `device`) belongs together with the code that adds the field masks for the fields around L498 (`if len(nsPaths) > 0 && config.NetworkServerEnabled {`) 

Alternatively we can put it together with the section around L521 (`// EUIs can not be updated, ...`)

Reply:
I agree it makes more sense there, but we would be overriding the value that comes from the passed arguments.

So afaik it has to be before `util.SetFields(&device, setEndDeviceFlags)`

Reply:
> Setting the "fallback" addresses from config (if still zero in `device`)

With that "if still zero in `device`" I meant `if device.XXXAddress == ""` checks